app/views/users/index.html.erb

```diff
@@ -1,6 +1,6 @@
-<h1>List of Users</h1>
-<ul>
-  <% @users.each do |user| %>
-    <li><%= user.name %></li>
-  <% end %>
-</ul>
+
+<% @users.each do |user| %>
+
+  <%= render 'shared/user_profile', user: user %>
+  
```

Comment:
Let's ensure our views conform to the wireframe design found on the [project instructions page](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md).

![image](https://github.com/adarachel/rails-blog/assets/12745474/c5f63f5a-3c87-4a58-bf4b-90d35293771a)